### DISCUSSION TITLE
Несколько вопросов.

### DISCUSSION THREAD

**Discussion by xBarselonx** [trickerer/Trinity-Bots#310] [Q&A]
Привет, trickerer!
1. После команды .npcbot command unbind <бот в таргет> бот возвращается на место респа до следующего призыва. Но после команды  .npcbot command rebind <бот в таргет> не всегда возвращается к игроку (возвращается только после перезагрузки сервера - 100%). 
2. После перезагрузки сервера не отображаются ники ботов (у меня это наблюдалось после .npcbot command unbind и дальнейшей перезагрузке (см. пункт 1). 
3. У фуривара не отображается второй меч на 70 уровне. Отображается только после обновления экипировки в меню сплетен. Может это только визуально, на урон не влияет?

**Comment by trickerer** [comment]
1. Не знаю, посмотрю.
2. Что значит не отображаются? 'Unknown' или что?
3. Для этого был фикс, возможно слетел после добавления трансмогрификации для ботов. На урон не влияет.

**Reply by xBarselonx** [reply]
Что значит не отображаются? 'Unknown' или что? - вообще не отображаются. Отображаются только после выделения самого бота. 


Еще забыл у пета бота-охотника под ником пишет «Null». Раньше этого не замечал. 

Да сразу не написал это АС.

**Reply by trickerer** [reply]
https://github.com/trickerer/Trinity-Bots/commit/48690e2dd8ea57dffd884615345888ea53998472

**Reply by trickerer** [reply]
Это по части клиента. После изменений в механизме владения ботов имена могут не отображаться. Включить их можно в настройках интерфейса Interface->Names->Friendly Units->Guardians (раньше было Pets)
То что у петов subname отображается как "NULL" это баг АС - если строка БД в значении Null то отображается так, если это "" то не отображается. Пока не буду менять. В АС полно подобных косяков именно в ядре.

**Reply by trickerer** [reply]
Посмотрел что там с фури варом. Похоже, то что есть сейчас это фикс и был. Раньше возможность восстановить показ друручного во второй руке была только снять и одеть заново. Это тоже на стороне клиента происходит. 2руч в левой руке - отказать, но правда в момент одевания всё обновляется.

**Comment by xBarselonx** [comment]
Возникла проблема
1. Временно увольняю всех ботов (.npcbot remove TARGET player)
2. Приглашаю обратно 4 бота танка, хила, ханта и рдд
3. Регаю рандомный инст героику лича
4. В итоге танк плетется где-то сзади, на мне висит щит земли (от шамана хила), хотя всегда висит на танке 
5. Возвращаюсь из инста, разбиваю пати, регаюсь опять - ничего не меняется. Но в рейде из 25 все норм.

**Reply by trickerer** [reply]
Это значит что в LFG ты зашёл в роли танка.

**Reply by xBarselonx** [reply]
Нет, в роли дд. Сегодня посмотрю еще раз.
